xclim/sdba/properties.py

```diff
@@ -340,9 +341,9 @@ def acf(
     xr.DataArray
       lag-{lag} autocorrelation of the variable over a {group.prop} and averaged over all years.
 
-    See also
+    See Also
     --------
-    statsmodels.tsa.stattools.acf
+    :doc:`generated/statsmodels.tsa.stattools.acf`
```

Comment:
The previous method (without `doc`) worked. How is the `:doc:` role different?

Reply:
Oh hey, you're right! I'm somewhat surprised. Will revert and read up on that. I was under the impression that we needed to add the `:<directive>:`, but I suppose not.

Reply:
I think it's a special feature of the `See also` section!